sdcm/tester.py

```diff
@@ -2119,6 +2120,23 @@ def run_fullscan_thread(fullscan_params: FullScanParams, thread_name: str):
 
         ).start()
 
+    def run_data_operations_thread(self, thread_name: str = "DataOperations"):
+        """Run thread of cql command select *
+
+        Calculate test duration and timeout interval between
+        requests and execute the thread with cqlsh command to
+        db node 'select * from ks.cf, where ks and cf are
+        random choosen from current configuration'
+
+        Keyword Arguments:
+            fullscan_params: FullScanParams object with params relevant for starting
+            the ScanOperationThread
+        """
+        DataOperationThread(
```

Comment:
test can stop this thread, in other cases we pass `self.db_cluster.nemesis_termination_event` into those thread classes, and they would stop when that event is set.